CONTRIBUTORS.md

```diff
@@ -23,3 +23,4 @@
 |19| [Peter Irvine](https://github.com/peirvine) | UMN | USA |
 |20| [Eric Moynihan](https://github.com/TooFiveFive) | UCC | Ireland |
 |21| [James Flynn](https://github.com/james-flynn-ie) | | Ireland |
+|22| [Tanmay Singhal](https://github.com/tanmaysinghal98) | | India |
```

Comment:
Please add this line only, no need to change the README. Also feel free to add your institution name. :) Let me know if you are facing any difficulty.